apps/teams-test-app/src/components/PeopleAPIs.tsx

```diff
@@ -14,9 +15,34 @@ const SelectPeople = (): React.ReactElement =>
   ApiWithTextInput<people.PeoplePickerInputs | undefined>({
     name: 'selectPeople',
     title: 'Select People',
-    onClick: async input => {
-      const result = input ? await people.selectPeople(input) : people.selectPeople();
-      return JSON.stringify(result);
+    onClick: {
+      validateInput: input => {
+        if (!input) {
+          return; //API allows for no input to be provided
+        }
+        return;
+      },
+      submit: {
+        withPromise: async input => {
+          const result = input ? await people.selectPeople(input) : people.selectPeople();
+          return JSON.stringify(result);
+        },
+        withCallback: (input, setResult) => {
+          const callback = (error: SdkError, people: people.PeoplePickerResult[]): void => {
+            if (error) {
+              setResult(JSON.stringify(error));
+            } else {
+              setResult(JSON.stringify(people));
+            }
+          };
+          if (input) {
+            people.selectPeople(callback, input);
+          } else {
+            people.selectPeople(callback);
+          }
+          return 'selectPeople()' + noHostSdkMsg;
```

Comment:
Do we still need to return something here after your change to update how `withCallback` is defined (https://github.com/OfficeDev/microsoft-teams-library-js/commit/fa1cd710d0db4484bf970730e3908024a82f41fc)?